$OpenBSD$

use += so that CFLAGS/LDFLAGS could be extended via the PORTS Framework

--- Makefile.orig	Thu Sep 25 14:22:23 2014
+++ Makefile	Thu Sep 25 14:22:30 2014
@@ -14,12 +14,12 @@ INSTALLDIR	:= $(DESTDIR)$(PREFIX)
 MANPREFIX		?= $(PREFIX)/share/man
 MANPREFIX		:= $(DESTDIR)$(MANPREFIX)
 
-CFLAGS		:= -Wall -Os -pedantic -std=c99 #-Werror -Wextra
+CFLAGS		+= -Wall -Os -pedantic -std=c99 #-Werror -Wextra
 CPPFLAGS  += -D_BSD_SOURCE -D_XOPEN_SOURCE=600
 CPPFLAGS  += -DNAME=\"$(TARGET)\" -DVERSION=\"$(VERSION)\" 
 CPPFLAGS	+= -DCOPYRIGHT=\"$(COPYRIGHT)\" -DLICENSE=\"$(LICENSE)\"
 
-LDFLAGS		:= -lX11 -lXrandr
+LDFLAGS		+= -lX11 -lXrandr
 
 all: $(TARGET)
 
